src/modules/UI/components/WalletListRow/WalletListTokenRow.ui.js

```diff
@@ -18,8 +17,8 @@ export type WalletListTokenRowOwnProps = {
 }
 
 export type WalletListTokenRowStateProps = {
-  currencyConverter: CurrencyConverter,
-  settings: Object
+  fiatValue: number,
+  cryptoAmount: number
```

Comment:
We now calculate the fiat value in the connector so that we do not have to pass the `currencyConverter` to the component